creator-node/scripts/run-tests.sh

```diff
@@ -50,7 +50,7 @@ run_unit_tests () {
 
```

Comment:
ik this isn't structure but pretty sure you meant to exclude any changes to this file before pushing?

Reply:
yep, just temporary